{"name": "test_wrong_pwd", "status": "passed", "description": "密码输入6次错误", "attachments": [{"name": "log", "source": "578ee2d9-f538-4460-9823-1575e425dfc4-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "0874a3fc-d97b-4a42-af40-bf4dcbbc9cdf-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "462dc4f6-abad-4772-bc81-b32fb2495640-attachment.txt", "type": "text/plain"}], "start": 1629283726676, "stop": 1629283727991, "uuid": "745e8b87-539f-45eb-b6ec-a935825f5ffe", "historyId": "7b9480ebb1961c7c14cb69cc7e65ba50", "testCaseId": "e4c5762512a76c681290f066bcc0e861", "fullName": "test_case.Login_case.TestLogin#test_wrong_pwd", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "登录模块"}, {"name": "story", "value": "参数填写不正确，登录失败"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "Login_case"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KSOAST3"}, {"name": "thread", "value": "8848-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.Login_case"}]}